templates/setup_envars.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# https://aka.ms/yaml
+parameters:
+- name: run_terraform
+  type: string
+- name: region
+  type: string
+
+steps:
+- bash: |
+    resource_group="tf$(date -u +'%m%dx%H%M%S')"
+    echo "##vso[task.setvariable variable=REGION]${{ parameters.region }}"
+    echo "##vso[task.setvariable variable=RESOURCE_GROUP_NAME]${resource_group}"
+    echo "##vso[task.setvariable variable=RUN_TERRAFORM_FILE]${{ parameters.run_terraform }}"
+    
+    echo "##vso[task.setvariable variable=ARM_SUBSCRIPTION_ID]${ARM_SUBSCRIPTION_ID}"
+    echo "##vso[task.setvariable variable=ARM_CLIENT_ID]${ARM_CLIENT_ID}"
+    echo "##vso[task.setvariable variable=ARM_CLIENT_SECRET]${ARM_CLIENT_SECRET}"
```

Comment:
I know these are already secure variables, so are scrubbed anyway. But since setting these values using vso syntax, it wouldn't hurt to add `isSecret=true` in case later the source of the variables moves away from a secure variable group to something else.

Reply:
The run did not like that: https://dev.azure.com/averevfxt/vfxt-github/_build/results?buildId=6355&view=logs&j=95dcbc90-489f-5462-2295-bdc9b9833630&t=a4fc1503-ffee-552d-b45f-3b546983f88a